src/pages/uxp-api/reference-js/Modules/fs/index.md

```diff
@@ -1,2 +1,401 @@
-# fs
-* [fs](fs.md)
+
```

Comment:
@kareemmyk  Please move the content from fs.md to index.md in torq-native

Reply:
This is a limitation of the source code for docs generation. Once we group a file into a folder, then a master ```index.md``` get's created by default
So at this stage the current state is our best option